docs/index.rst

```diff
@@ -0,0 +1,12 @@
+Python client for the Yola API
+==============================
+
+.. toctree::
+   :maxdepth: 1
+
```

Comment:
Yeah, I think the title "Python client for the Yola API" is pretty much all we can describe it as for now.